app/views/TermsAndConditions.scala.html

```diff
@@ -15,31 +15,45 @@
  *@
 
 @import config.FrontendAppConfig
+@import models.{RegistrationReason, TakingOverBusiness, ChangingLegalEntityOfBusiness}
 
 @this(
-    layout: views.html.layouts.layout,
-    h1: components.h1,
-    p: components.p,
-    bullets: components.bullets,
-    formWithCsrf: FormWithCSRF,
-    button: components.button
+        layout: views.html.layouts.layout,
+        h1: components.h1,
+        p: components.p,
+        bullets: components.bullets,
+        formWithCsrf: FormWithCSRF,
+        button: components.button
 )
 
-@()(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+@(regReason: Option[RegistrationReason])(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
 @layout(pageTitle = Some(titleNoForm(messages("termsAndConditions.heading"))), backLink = true) {
 
     @h1("termsAndConditions.heading")
 
-    @p{ @messages("termsAndConditions.para") }
+    @p {
+        @messages("termsAndConditions.para")
+    }
 
-    @bullets(
-        "termsAndConditions.bullet1",
-        "termsAndConditions.bullet2",
-        "termsAndConditions.bullet3",
-        "termsAndConditions.bullet4",
-        "termsAndConditions.bullet5"
-    )
+    @if(Seq(TakingOverBusiness).exists(regReason.contains)) {
```

Comment:
can just do regReason.contains(TakingOverBusiness)
the Seq().exists(_.contains) pattern is only needed when we wanna check if _ contains one of multiple options.